blog/app/models/article.rb

```diff
@@ -2,4 +2,5 @@ class Article < ActiveRecord::Base
 	has_many :comments, dependent: :destroy
 	validates :title, presence: true,
                     length: { minimum: 5 }
+  validates :text, presence: true
```

Comment:
its gonna suck but can you go through and change all of your files to use spaces instead of tabs? 

should just need to open each one and do Sublime Menu > View > Indentation > Convert Indentation to Tabs

Or possibly just do CMD+SHIFT+F and find replace all tabs with spaces in *.rb files